addons/godot-firebase/auth/auth.gd

```diff
@@ -379,7 +379,7 @@ func _on_FirebaseAuth_request_completed(result : int, response_code : int, heade
         var json_result = JSON.parse(bod)
         if json_result.error != OK:
             Firebase._printerr("Error while parsing auth body json")
-            emit_signal("auth_request", "Error while parsing auth body json", json_result)
+            emit_signal("auth_request", ERR_PARSE_ERROR, "Error while parsing auth body json")
```

Comment:
https://github.com/LeonardMeagher2/GodotFirebase/blob/4a0eebdc433355e800467a8465ec93e007180b8b/addons/godot-firebase/auth/auth.gd#L14

The documentation for this signal says error should have error codes and a message